map/place_page_info.hpp

```diff
@@ -22,8 +24,22 @@ enum class SponsoredType
   Geochat
 };
 
-namespace place_page
+struct Banner
 {
+  explicit Banner(feature::Metadata const & metadata);
+
+  /// Returns true, if the current time is within the working range of the banner.
+  bool IsActive() const;
+
+  string TitleId() const;
+  string MessageId() const;
+  string IconId() const;
+  string Url() const;
+
+private:
+  feature::Metadata const & m_metadata;
```

Comment:
Запрет на копирование поломал android сборку:

jni/com/mapswithme/maps/Framework.cpp:466:10: error: object of type 'place_page::Info' cannot be assigned because its copy assignment operator is implicitly deleted
  m_info = info;
         ^